README.md

```diff
@@ -25,6 +25,23 @@ See the [Electron docs for using native modules][electron-native-modules] to ens
 [electron-rebuild]: https://github.com/electron/electron-rebuild
 [electron-native-modules]: https://www.electronjs.org/docs/tutorial/using-native-node-modules
 
+### Known problems using with electron:
+
+- If you try to use `usb-detection` with [electron](https://www.electronjs.org/), you run into the following error. You can follow the steps below to resolve the problem.
+    ```
+    detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing
+    ```
+
+    You can solve this problem following next steps:
+
+    1. If you have not installed electron-rebuild just install it with the command: `npm i -D electron-rebuild`
+    1. Remove from the `node_modules` folder the `usb-detection` folder.
+    1. Remove the file `package-lock.json`, this is suggested for ensure a clean installation, but you can try first without remove it.
+    1. Run `npm i` to install non-installed modules.
+    1. And finally run `./node_modules/.bin/electron-rebuild`
+
+    It is very important to run `./node_modules/.bin/electron-rebuild` directly after `npm i`. This is important because `usb-detection` must be installed to be rebuilt for electron target version.
+
```

Comment:
I think we can simplify this down to:

```
If you run into the following error, here are the exact steps you can use:
\```
detection.node was compiled against a different Node.js version using NODE_MODULE_VERSION 72. This version of Node.js requires NODE_MODULE_VERSION 80. Please try re-compiling or re-installing
\```

 1. `npm i electron-rebuild --save-dev`
 1. `./node_modules/.bin/electron-rebuild`
```

Reply:
@mcous does this look accurate?

Reply:
@mcous Thanks for checking over this!

Reply:
Ok guys, I going to follow their advice and update the PR. Thanks for advice me!